kornia/contrib/extract_patches.py

```diff
@@ -107,7 +107,8 @@ class CombineTensorPatches(nn.Module):
 
     Args:
         patches: patched tensor.
-        window_size: the size of the sliding window and the output patch size.
+        original_size: the size of the original tensor and the output patch size.
```

Comment:
The docs above must be updated, it mentions padding, stride, etc. Also, worth to cross link a sentence saying that this method it’s recommended to be used after extract patches. See: https://github.com/kornia/kornia/blob/master/kornia/geometry/transform/imgwarp.py#L84

Reply:
Docs have been updated.